pkg/server/local_observer.go

```diff
@@ -220,10 +220,10 @@ func (s *LocalObserverServer) GetFlows(
 	req *observer.GetFlowsRequest,
 	server observer.Observer_GetFlowsServer,
 ) (err error) {
-	return getFlows(req, server, s)
+	return s.getFlows(req, server, s)
 }
 
-func getFlows(
+func (s *LocalObserverServer) getFlows(
```

Comment:
Why does this function need to become a `LocalObserverServer` method?

Reply:
Because access to `s.opts` is required to retrieve the list of `OnBuildFilter`:

```
        filterList := append(filters.DefaultFilters, s.opts.OnBuildFilter...)
```

Reply:
Restructured so this receiver is no longer required.